docs/framework/install/on-windows-8-1.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Install the .NET Framework on Windows 8.1
+ms.custom: "updateeachrelease"
+description: Learn how to install .NET Framework on Windows 8.1
+author: rlander
+ms.author: mairaw
+keywords: .NET Framework, Install
+ms.date: 10/22/2017
+ms.topic: article
+ms.prod: .net-framework
+ms.devlang: dotnet
+---
+
+# Install the .NET Framework on Windows 8.1
+
+The .NET Framework is required to run many applications on Windows. You can use the following instructions to install it. You may have arrived on this page after trying to run an application and seeing the following dialog on your machine.
+
+![This application could not be started](./media/this-application-could-not-be-started.png)
+
+These instructions will help you install the .NET Framework versions you need. The latest version is often your best choice. The latest version is the .NET Framework 4.7.1.
+
+## .NET Framework 4.7.1
+
+* [Download the .NET Framework 4.7.1](https://www.microsoft.com/net/framework/versions/net471?utm_source=ms-docs&utm_medium=referral)
+
+The [.NET Framework 4.7.1](https://www.microsoft.com/net/framework/versions/net471?utm_source=ms-docs&utm_medium=referral) is included with the [Windows 10 Fall Creators Update](https://www.microsoft.com/software-download/windows10) and [Windows Server 2016 Version 1709](https://docs.microsoft.com/windows-server/get-started/get-started-with-1709).
```

Comment:
I find it confusing that, in a Windows 8.1 topic, there's a paragraph about the Windows 10 versions on which the framework is pre-installed.